glide confusion w.r.t. repository

Glide and the go imports all seem to think this repository is at github.com/uber, not jaegertracing.

I'm not enough of a go expert to understand all the tradeoffs here, but it has lead me into a number of weird build time issues with vendor packages not found and the like - would there be any objections to a mass s/r to update this to the actual home?

example errors:

```
$ glide install
[WARN]  The name listed in the config file (github.com/uber/jaeger) does not match the current location (.)
[INFO]  Downloading dependencies. Please wait...
....

$ make build_examples
go build -o ./examples/hotrod/hotrod-demo ./examples/hotrod/main.go
examples/hotrod/main.go:8:2: cannot find package "github.com/uber/jaeger/examples/hotrod/cmd" in any of:
        /usr/lib/go-1.6/src/github.com/uber/jaeger/examples/hotrod/cmd (from $GOROOT)
        /home/robertc/go/src/github.com/uber/jaeger/examples/hotrod/cmd (from $GOPATH)
Makefile:96: recipe for target 'build_examples' failed

$ make
cmd/agent/main.go:20:2: cannot find package "github.com/pkg/errors" in any of:
        /usr/lib/go-1.6/src/github.com/pkg/errors (from $GOROOT)
        /home/robertc/go/src/github.com/pkg/errors (from $GOPATH)

$ ls vendor/github.com/pkg/errors/ -ld
drwxr-xr-x 2 robertc robertc 4096 Oct  2 18:43 vendor/github.com/pkg/errors/
```

I do have the vendor experiment enabled:
```
$ go env
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/robertc/go"
GORACE=""
GOROOT="/usr/lib/go-1.6"
GOTOOLDIR="/usr/lib/go-1.6/pkg/tool/linux_amd64"
GO15VENDOREXPERIMENT="1"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0"
CXX="g++"
CGO_ENABLED="1"
```

--- a/README.md
+++ b/README.md
@@ -45,6 +45,10 @@ Jaeger is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
  * [UI](https://github.com/uber/jaeger-ui)
  * [Data model](https://github.com/uber/jaeger-idl)
 
+## Building From Source
+
+See [CONTRIBUTING](./CONTRIBUTING.md).
+
 ## Contributing
 
 See [CONTRIBUTING](./CONTRIBUTING.md).

